ansible/provision.yaml

```diff
@@ -1,4 +1,24 @@
 ---
+- hosts: all
+  name: Prepare Ansible - Flatcar Python setup
+  become: true
+  gather_facts: false
+  tasks:
+    - name: Flatcar no update
+      changed_when: false
+      script: ../packer/files/no-update-flatcar.sh
+    - name: exec bootstrap python
+      changed_when: false
+      script: ../packer/files/bootstrap-flatcar.sh
+    - name: check for flatcar python
+      raw: stat /opt/bin/.bootstrapped
+      changed_when: false
+      failed_when: false
+      register: flatcar_bootstrapped
+    - name: change python if bootstrapped
+      set_fact:
+        ansible_python_interpreter: /opt/bin/python
+      when: flatcar_bootstrapped.rc == 0
```

Comment:
hey @jr0d this is what I did about python bootstrapping.

I also had to do a small change to the bootstrap script to use `/etc/os-release` https://github.com/mesosphere/konvoy-image-builder/pull/39/files#diff-d4c832f60ff555a5199070ac746ff4f8e43ff43eaeccc502ce53c3723afc510cR5-R7